dockerimages/terminal/latest/html/src/template.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <title>
+        <%= htmlWebpackPlugin.options.title %>
+    </title>
+    <link inline rel="icon" type="image/png" href="favicon.png">
+    <% for (const css in htmlWebpackPlugin.files.css) { %>
+        <link inline rel="stylesheet" type="text/css" href="<%= htmlWebpackPlugin.files.css[css] %>">
+        <% } %>
+</head>
+
+<body>
+    <script type="text/javascript">
+        window.addEventListener("message", (e) => {
+            if (e.data.command) {
```

Comment:
do we need to check command type? eg: run/close/new

Reply:
ttyd does not need to judge, terminal app needs to judge

Reply:
Does ttyd need to judge the source of the request?